### PR TITLE
created a hero section

### PR DESCRIPTION
**Pull request by rcode321**
Created a Hero Section of freeCodeCamp Manila landing page
here is some screenshot of before and after the project.

before
after

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css">
+
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css">
+  <title>freeCodeCamp Manila</title>
+</head>
```
Comment (jromest) [inline src/index.html]: You have duplicated `head` tag, see line 32

File: src/index.html [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+    <p>We are a local community and study group learning to code through freeCodeCamp.org</p>
+    <a href="https://www.freecodecamp.org/" target="_blank" class="button">Join the community</a>
+    <div class="socilamediaMileston">
+      <a href="https://www.facebook.com/groups/free.code.camp.manila/" target="_blank"><i
+          class="fab fa-facebook-square fa-2x"></i></a>
+      <span>4000+ Members, 1000+ Likes</span>
+    </div>
+    <img id="hero-image" src="images/hero.jpg" alt="">
+  </div>
+</body>
```
Comment (jromest) [inline src/index.html]: You also have duplicated `body` tag, add the hero section before the Join the Community section

File: src/index.html [hunk 3]
```diff
@@ -1,5 +1,34 @@
...
+
+</html>
```
Comment (jromest) [inline src/index.html]: Please remove closing `html` tag

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,154 @@
+body {
```
Comment (jromest) [inline src/style.css]: There's a CSS file under `src/css/style.css`, please add these styles to that file

Also, please use class selector, you might wanna read about BEM convention

File: src/index.html (hunk 1 above)
Comment (rcode321) [reply src/index.html]: I update the src/index.html and remove the duplicated head tag

File: src/index.html (hunk 3 above)
Comment (rcode321) [reply src/index.html]: I remove also the closing html tag.

File: src/index.html (hunk 1 above)
Comment (jromest) [reply src/index.html]: Hey @rcode321, I think you have removed the wrong head tag the current styles have been replaced.

Here's a screenshot
localhost_1234-freeCodeCampManila

File: src/style.css (hunk 4 above)
Comment (jromest) [reply src/style.css]: We're not strict BEM convention, just use class selectors 😄

**Comment by jromest** [comment]
Hi. I'm closing this as it is now a deleted fork repo where the PR comes from. If you want to make the suggested changes you are welcome to open a new PR. Thanks!

**Pull request closed without merging**